lighthouse-core/test/gather/computed/first-interactive-test.js

```diff
@@ -28,7 +28,7 @@ describe('FirstInteractive computed artifact:', () => {
     return computedArtifacts.requestFirstInteractive({traceEvents: tooShortTrace}).then(() => {
       assert.ok(false, 'should have thrown for short trace');
     }).catch(err => {
-      assert.equal(err.message, 'trace not at least 5 seconds longer than FMP');
+      assert.ok(err.message, 'FMP_TOO_LATE_FOR_FCPUI');
```

Comment:
Ah, separate note on this line.. seems like it should still be using `equal()` ;)

Reply:
haha oops, done :)